lib/fastlane/plugin/wpmreleasetoolkit/actions/common/get_prs_list_action.rb

```diff
@@ -35,23 +35,22 @@ def self.return_value
 
       def self.details
         # Optional:
-        'Generate the list of the PRs from `start_tag` to `end_tag`'
+        'Generate the list of the PRs in the given `repository` for the given `milestone` at the given `report_path'
```

Comment:
I wonder if we can call `description` here? To DRY, given there's not difference.

Reply:
I might be wrong but I feel like fastlane could even be smart enough to fallback to `self.description` when it prints the action's documentation if `self.details` is empty 🤔

Reply:
For now, it's explicit, but we can always change it in the future 🤷

Reply:
👍